tests/test_app.py

```diff
@@ -612,10 +624,11 @@ async def test_list():
             assert len(file_names) == 4
 
 
+@settings(deadline=None)
```

Comment:
The auth client service check makes these take longer now. Didn't seem necessary to keep the deadline that was breaking hypothesis. See also below.

Reply:
I don't understand the context here - what's setting the deadline?

Reply:
`deadline` is a thing that the `hypothesis` package uses for test timing. If they take too long to run, they fail. Default's 200ms. I've found hypothesis, at least how it's implemented here, to be on the unpredictable side, especially when you throw in disk IO and HTTP calls. There's at least one other test that just removes the deadline, I figured it was appropriate here, too.

Reply:
Could you add comments to the tests as to why the deadline is removed to help out the next poor sucker that has to work on this?

Reply:
Done.